src/autoUnmount.js

```diff
@@ -0,0 +1,13 @@
+import render from './render';
+
+export default function autoUnmount() {
+  let queue;
+  beforeEach(() => {
+    queue = render.unmountQueue = [];
+  });
+  afterEach(() => {
+    render.unmountQueue = null;
+    queue.forEach(unmount => unmount());
+    queue.length = 0;
+  });
```

Comment:
This lets us call `autoUnmount` within a `describe` block to enable auto-unmounting for a subset of tests.

You can also call it from the top-level if you want auto-unmounting for all tests in the file.